ablateLibrary/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ else()
 endif()
 
 ### Ablate Library Setup ###
-add_library(ablateLibrary SHARED "")
+add_library(ablateLibrary SHARED "" flow/boundaryConditions/NeumannGhost.cpp)
```

Comment:
Would you mind reverting this line.  It doesn't look like you are adding any boundary conditions in this PR.